src/interface/retrieve-order.interface.ts

```diff
@@ -0,0 +1,59 @@
+import { ICustomer } from '../interface';
+import { PaymentTypeEnum, ServiceTypeEnum } from '../enum';
+
+export interface IPayload {
+    customer: ICustomer;
+    info: IRetrieveInfo;
+    items: IRetrieveItem[];
+    configuration: {};
```

Comment:
what does `{}` mean? 
is it always empty?

Reply:
I dont know in swagger schema is not declared, but on server response is empty.

Reply:
better use `configuration: Record<string, any>`;